qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -85,17 +86,18 @@ def __init__(
                 - if ``quantum_kernel`` is passed and ``precomputed`` is set to ``True``. To use
                 a precomputed kernel, ``quantum_kernel`` has to be of the ``None`` type.
             TypeError:
-                - if ``quantum_instance`` neither instance of ``QuantumKernel`` nor ``None``.
+                - if ``quantum_kernel`` neither instance of
+                  :class:`~qiskit_machine_learning.kernels.BaseKernel` nor ``None``.
         """
 
         if precomputed:
             if quantum_kernel is not None:
                 raise ValueError("'quantum_kernel' has to be None to use a precomputed kernel")
         else:
             if quantum_kernel is None:
-                quantum_kernel = QuantumKernel()
-            elif not isinstance(quantum_kernel, QuantumKernel):
-                raise TypeError("'quantum_kernel' has to be of type None or QuantumKernel")
+                quantum_kernel = FidelityQuantumKernel()
+            elif not isinstance(quantum_kernel, BaseKernel):
+                raise TypeError("'quantum_kernel' has to be of type None or BaseKernel")
```

Comment:
Typically we do not police types like this especially when only one type is specified by the typehint. If you wish to keep it then it might be nice to indicate the failing type in the error message too.

Reply:
Removed this `elif` clause.